### PR TITLE
Create main.py

### PR DESCRIPTION
**Pull request by mgracer48**
@DissonancePM wanted a hello world python file

### CODE REVIEW

**Comment by DissonancePM** [comment]
we will have to look this over please wait a couple of weeks

**Comment by DissonancePM** [comment]
oh wait sorry this went through travis and the build failed

File: main.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print "hello world"
```
Comment (DissonancePM) [inline main.py]: there seems to be an assertion error on this line

**Comment by loblao** [comment]
Unfortunately this does not run on my Tizen phone and we can't accept platform-specific code.

File: main.py (hunk 1 above)
Comment (mfwass) [inline main.py]: This does not have Python 3.4 support.

**Comment by DissonancePM** [comment]
@loblao I believe we should lock this conversation so that people don't try to refute the facts.

**Pull request closed without merging**